src/CMakeLists.txt

```diff
@@ -155,6 +155,13 @@ if (TILES)
     if (RELEASE)
         install(TARGETS cataclysm-tiles RUNTIME)
     endif ()
+
+    if (TARGET ZLIB::ZLIB)
+        target_link_libraries(cataclysm-tiles-common PUBLIC ZLIB::ZLIB)
+    else()
+        target_link_libraries(cataclysm-tiles-common PUBLIC ${ZLIB_LIBRARIES})
+    endif()
```

Comment:
Would the `else` branch ever be taken? We have `find_package(ZLIB, REQUIRED)` at root CMakeLists, so we always have the ZLIB target defined, right?

Reply:
Yes but `ZLIB::ZLIB` may not be defined and only ZLIB_* variables would be available on such systems.